tests/integration/istio-injection/istioinjection/fixture.go

```diff
@@ -0,0 +1,241 @@
+package istioinjection
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/avast/retry-go"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "k8s.io/api/core/v1"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const HTTPDBSERVICE_IMAGE = "eu.gcr.io/kyma-project/example/http-db-service:0.0.6"
+
+func (testSuite *TestSuite) createNamespace() error {
+	log.Infof("Creating namespace '%s", testSuite.namespace)
+	err := retry.Do(func() error {
+		_, err := testSuite.k8sClient.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: testSuite.namespace,
+			},
+			Spec: v1.NamespaceSpec{},
+		})
+		return err
+	},
+		retry.Attempts(5),
+		retry.Delay(3*time.Second))
+	if err != nil {
+		log.Errorf("Cannot create namespace '%s': %v", testSuite.namespace, err)
+	}
+	return err
+}
+
+func (testSuite *TestSuite) deleteNamespace() error {
+	log.Infof("Deleting namespace '%s", testSuite.namespace)
+	var deleteImmediately int64
```

Comment:
Please explicitly declare the variable as 0. Sure, we all know zero values, but this does have an impact on readability.